packages/esm-patient-medications-app/src/types/order-basket-item.ts

```diff
@@ -1,10 +1,4 @@
-import {
-  DosingUnit,
-  MedicationDosage,
-  MedicationFrequency,
-  MedicationRoute,
-  OrderTemplate,
-} from '../api/drug-order-template';
+import { DosingUnit, MedicationFrequency, MedicationRoute, OrderTemplate } from '../api/drug-order-template';
```

Comment:
We could get rid of the `MedicationDosage` type if it's no longer in use.

Reply:
Sure!